c++/src/Writer.cc

```diff
@@ -38,9 +38,10 @@ namespace orc {
     FileVersion fileVersion;
     double dictionaryKeySizeThreshold;
     bool enableIndex;
+    RleVersion rleVersion;
 
     WriterOptionsPrivate() :
-                            fileVersion(0, 11) { // default to Hive_0_11
+                            fileVersion(0, 12) { // default to Hive_0_12
```

Comment:
We should use the static constants proposed in PR https://github.com/apache/orc/pull/274 moving forward.

Reply:
I can do that after PR #274 checked in